app/mailers/participant_mailer.rb

```diff
@@ -0,0 +1,44 @@
+class ParticipantMailer < ApplicationMailer
+
+    def send_reminder(interview_id,time)
+        # puts "time from mailer"
+        # puts time
+        @interview = Interview.find(interview_id)
+        # puts @interview.updated_at
+        # puts "updated time"
+        if(@interview.updated_at <= time)
+            # puts "Yass"
+            @interviewees = Interviewee.where("id in (?)", @interview.interviewee_ids)
```

Comment:
Interviewee.where(:id => @interview.interviewee_ids)

Reply:
changed in the commit 
64a01bd